typing/predef.ml

```diff
@@ -468,6 +468,8 @@ let add_small_number_extension_types add_type env =
   |> add_type ident_unboxed_float32 ~jkind:Jkind.Const.Builtin.float32
 
 let or_null_kind tvar =
+  (* CR layouts v3: use [Variant_with_null] when it's supported
```

Comment:
```suggestion
  (* CR layouts v3.0: use [Variant_with_null] when it's supported
```

Reply:
It's fixed in the final PR, not worth editing at this point.